tests/dpsdgym/main.py

```diff
@@ -1,52 +1,34 @@
-import sys
-import os
-import time
-import mlflow
-import json
+import argparse
 
+import benchmark
 import conf
+import mlflow
+
+def _parse_args():
+    parser = argparse.ArgumentParser(prog="DPSDGym", 
+                                    description="Differentially private synthetic data generators benchmarking suite", 
+                                    epilog="Sample command: python main.py -d bank adult -e 0.01 0.1 1 -m pmse ml_eval wasserstein rsa -r test_run")
 
-from load_data import load_data
+    parser.add_argument('-d', '--dataset', nargs="+", default=conf.KNOWN_DATASETS, help="Datasets names on which the benchmarks will be executed")
+    parser.add_argument('-e', '--epsilon', nargs="+", default=conf.EPSILONS, help="Epsilons values for which the models will be evaluated")
+    parser.add_argument('-m', '--metric', nargs="+", default=conf.KNOWN_METRICS, help="Differential privacy metrics for which the models will be evaluated")
+    parser.add_argument('-r', '--run', default="dpbench_run", help="Run name for mlflow")
+    parser.add_argument('-p', '--path', default="results/", help="Path of the folder to save the results")
 
-from synthesis import run_all_synthesizers
-from evaluate import run_ml_eval, run_wasserstein, run_pMSE
+    args = parser.parse_args()
+    if isinstance(args.dataset[0], str) and len(args.dataset)==1:
+        args.dataset = args.dataset[0].split()
```

Comment:
This is ok for now but comma separated strings will probably be better. Dataset names are more likely to have a space than a comma